snakebite/client.py

```diff
@@ -393,6 +394,60 @@ def _handle_rename(self, path, node, dst):
         response = self.service.rename(request)
         return {"path": path, "result": response.result}
 
+
+    def rename_dont_move(self, path, dst, overwriteDest=False):
```

Comment:
I think it's intuitive. You rename but you don't move. It's explicit.

However, I think it might as well be called `rename2`, as it also has the `overwriteDest` parameter.